.travis.yml

```
@@ -5,13 +5,49 @@ node_js:
   - '8'
   - '6'
 install:
+  - 'if [ -n "${JEST}" ]; then yarn add --dev "jest@${JEST}"; fi'
```

```suggestion
  - 'if [ -n "${JEST}" ]; then yarn add --dev "jest@${JEST}" "babel-jest@${JEST}"; fi'
```

Not sure how that cooperates with babel 6 vs 7, though... I'm unsure if it's feasible to test with more than 24 and 25, as those are the only versions that support babel 7 out of the box. With 22 and 23 we'll also need `babel-core@bridge` (not a biggie, so I guess you can try adding it as well). Jest 21 doesn't support babel 7 at all, so that's a no-go

---

I've updated the build matrix to run jest 24-25 x eslint 4-6 x node 6-12 for now. PR should be in a good place unless you have other suggestions.